index.html

```diff
@@ -74,259 +73,263 @@ <h2 class="h2 color-dark">
 			</section>
 
 			<!-- WORKS SECTION -->
+			<div id="portfolio">
+				<div class="container">
+					<div class="portfolio padding-5 grid justify-center">
+						<!-- PORTFOLIO CARD NUMBER 1 -->
 
-			<div class="section" id="portfolio">
-				<div class="portfolio padding-5 grid justify-center">
-					<!-- PORTFOLIO CARD NUMBER 1 -->
-					<article class="portfolio__card grid">
-						<header class="portfolio__header">
-							<img src="images/portfolio/work_1.png" alt="Work 1" class="portfolio__image" />
-						</header>
-						<div class="portfolio__content grid">
-							<h3 class="portfolio__title color-dark">Tonic</h3>
-							<div class="portfolio__profile flex-row align-center">
-								<h4 class="uppercase color-info">Canopy</h4>
-								<ul class="portfolio__list flex-row">
-									<li class="portfolio__item">Back End Dev</li>
-									<li class="portfolio__item">2015</li>
+						<article class="portfolio__card grid">
+							<header class="portfolio__header">
+								<img src="images/portfolio/work_1.png" alt="Work 1" class="portfolio__image" />
+							</header>
+							<div class="portfolio__content grid">
+								<h3 class="portfolio__title color-dark">Tonic</h3>
+								<div class="portfolio__profile flex-row align-center">
+									<h4 class="uppercase color-info">Canopy</h4>
+									<ul class="portfolio__list flex-row">
+										<li class="portfolio__item">Back End Dev</li>
+										<li class="portfolio__item">2015</li>
+									</ul>
+								</div>
```

Comment:
- [ ] You've done an amazing job on your cards! I just wanted to offer a small suggestion to help them align even better with the [provided template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1). If you wouldn't mind, could you please **take a look at the circle icon between your elements in the template and consider adding them?** It would add a nice touch to the overall design. 🙋‍♂️

- [ ] Additionally, would you please take a moment to check **the spacing between your image and text?** Making sure they are properly aligned **will greatly improve the visual appeal.** 🙏**_This is a very small change, just a few pixels._** 

   ### Your design:
   ![image](https://github.com/Rysth/Portfolio-Project/assets/107952724/4bd0be09-bb0f-4213-a723-b83246216393)

   ### Figma design:
   ![image](https://github.com/Rysth/Portfolio-Project/assets/107952724/7de06bf8-2942-49c9-9b96-61f3f6045f0f)
   
   _Just a friendly reminder that following the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122) is a key requirement for this project. You're doing a great job so far, and it's important to keep the template in mind as you continue with your design to ensure that your project meets all of the project requirements._
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/9650a956-df52-488a-b9c1-dd120581b502)